zulip_bots/zulip_bots/bots/idonethis/test_idonethis.py

```diff
@@ -0,0 +1,90 @@
+from unittest.mock import patch
+
+from zulip_bots.test_lib import BotTestCase
+
+import requests
+
+class TestIDoneThisBot(BotTestCase):
+    bot_name = "idonethis"  # type: str
```

Comment:
Can you use python3-sytle annotations?

Reply:
I understand that the variable annotation syntax is pep 526, which is implemented in python 3.6. When I try to use that syntax I get the following error (in travis too):
```
    bot_name: str = "idonethis"
            ^
SyntaxError: invalid syntax
```
The Travis log also shows
```
$ python --version
Python 3.4.6
```
Which means that the python version is below the required 3.6. Should I fall back to the type in comments?

Reply:
Hmm, yeah, let's keep the comment annotation then.